### PR TITLE
no description....kalimut ko unsa ni..

### PR DESCRIPTION
**Pull request by darksidebug**

### CODE REVIEW

File: app/Classes/Facades/StringHelper.php [hunk 1]
```diff
@@ -9,6 +9,8 @@ public static function toReadableStatus($value){
         $data = [
             'pending_building_inspection' => 'Pending for Building Inspection',
             'pending_waterworks_inspection'=>'Pending for Waterworks  Inspection',
+            'pending_for_payment'=>'Pending for Payments',
+            'pending' => 'Pending',
```
Comment (armanmasangkay) [inline app/Classes/Facades/StringHelper.php:13]: Remove this array 'pending' is no longer necessary

File: app/Http/Controllers/ServicesPaymentController.php [hunk 2]
```diff
@@ -31,6 +34,31 @@ public function search(Request $request)
...
-        return view('pages.cashier-services-transaction-payment', ['services' => $services, 'route' => 'admin.services-payment-search']);
+        return view('pages.users.cashier.services-transaction-payment', ['services' => $services, 'route' => 'admin.services-payment-search']);
+    }
+
+    public function savePayment(Request $request)
+    {
+        $validator = Validator::make($request->all(),[
+            'orNum' => 'required|unique:payments,or_no',
+            'inputedAmount' => 'required|numeric|gt:0'
+        ]);
```
Comment (armanmasangkay) [inline app/Http/Controllers/ServicesPaymentController.php:45]: Make this as a 'Form Request'

File: app/Http/Controllers/ServicesPaymentController.php [hunk 3]
```diff
@@ -31,6 +34,31 @@ public function search(Request $request)
...
+
+        if($validator->fails()){
+            return response()->json(['created' => false, 'errors' => $validator->errors()]);
+        }
```
Comment (armanmasangkay) [inline app/Http/Controllers/ServicesPaymentController.php:49]: Move this to the form request

**Review by armanmasangkay — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/ServicesPaymentController.php (hunk 3 above)
Comment (darksidebug) [reply app/Http/Controllers/ServicesPaymentController.php:49]: humana te...

File: app/Classes/Facades/StringHelper.php (hunk 1 above)
Comment (darksidebug) [reply app/Classes/Facades/StringHelper.php]: nahuman na ni te?

File: app/Http/Controllers/ServicesPaymentController.php (hunk 2 above)
Comment (darksidebug) [reply app/Http/Controllers/ServicesPaymentController.php]: gibalhin na ni te...

**Pull request merged**